core/gesture.ts

```diff
@@ -326,14 +301,12 @@ export class Gesture {
     if (!this.targetBlock_) {
       return false;
     }
-
     if (this.flyout_) {
```

Comment:
Update TSDoc: "Update this gesture to record whether a block is being dragged" --> "Check whether to start a block drag. If a block should be dragged, either from the flyout or in the workspace, create the necessary BlockDragger and start the drag."